### PR TITLE
further README improvements

### PR DESCRIPTION
**Pull request by ofrobots**
@matthewloring ptal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,22 +6,19 @@
...
 ## Prerequisites
-* You are using Node.js 0.12+ for your application. (Node.js v5+ is recommended).
-* The source of your application is uploaded to your cloud project's [source repository](https://cloud.google.com/tools/cloud-repositories/docs/). The Debugger UI needs the source to be available in order to set breakpoints.
+* Your application will need to be using Node.js version 0.12 or greater. Node.js v5+ is recommended.
+* The source of your application is uploaded to a [cloud source repository](https://cloud.google.com/tools/cloud-repositories/docs/). The Debugger UI needs the source to be available in order to set breakpoints.
 
 ## Quick Start (Node.js v4.x+)
 ```shell
-# Install module, and save it as a dependency
+# Install with `npm` or add to your `package.json`.
```
Comment (matthewloring) [inline README.md]: s/or/and/

File: README.md [hunk 2]
```diff
@@ -30,11 +27,11 @@ npm install --save @google/cloud-debug
      "start": "node --require @google/cloud-debug server.js"
    ...
 ```
-Navigate to the Debug tab within the [Google Developers Console][dev-console] to set breakpoints and start debugging.
+Deploy your application, navigate to the [Debug tab][debug-tab] within the [Google Developers Console][dev-console] to set breakpoints and start debugging.
```
Comment (matthewloring) [inline README.md]: ... application, and navigate ...

File: README.md [hunk 3]
```diff
@@ -30,11 +27,11 @@ npm install --save @google/cloud-debug
...
 
-## Detailed instructions
+## Installation
 
-1. Install the debug agent module.
+1. Install the debug agent module via `npm` or add it to your `package.json`.
```
Comment (matthewloring) [inline README.md]: s/or/and

File: README.md [hunk 4]
```diff
@@ -51,24 +48,47 @@ Navigate to the Debug tab within the [Google Developers Console][dev-console] to
...
+
+### Google App Engine Managed VMs
+
+If you are using [Google App Engine Managed VMs](https://cloud.google.com/appengine/docs/managed-vms/), you do not have to do any additional configuration.
+
+### Google Compute Engine
+
+Your VM instances need to be created with `cloud-platform` scope if created via [gcloud](https://cloud.google.com/sdk) or the 'Allow API access' checkbox selected if created via the [console](https://console.developers.google.com) (see screenshot).
+
+![GCE API](doc/images/gce.png?raw=true)
```
Comment (matthewloring) [inline README.md]: Does this image exist in this repo?

File: README.md [hunk 5]
```diff
@@ -51,24 +48,47 @@ Navigate to the Debug tab within the [Google Developers Console][dev-console] to
...
+
+### Google Container Engine
+
+Container Engine nodes need to also be created with the `cloud-platform` scope, which is configurable during cluster creation. Alternatively, you can follow the instructions for using a service account under [running elsewhere](#running-elsewhere). It's recommended that you store the service account credentials as [Kubernetes Secret](http://kubernetes.io/v1.1/docs/user-guide/secrets.html).
+
+## Running elsewhere
+
+If your application is running outside of Google Cloud Platform, such as locally, on-premise, or on another cloud provider, you can still use Cloud Debug.
+
+1. You will need to specify the numeric project ID of your project. Your numeric project ID is visible in [Google Cloud Platform Project](https://console.developers.google.com/project?_ga=1.258049870.576536942.1443543237), it may be something like `1002377379474884`.
```
Comment (matthewloring) [inline README.md]: Looks like the preferred wording here is console: see https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/171/files

File: README.md (hunk 1 above)
Comment (ofrobots) [inline README.md]: This is cf. https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/blob/master/README.md.

Comment (ofrobots) [inline README.md]: I see the way you are interpreting it, but assuming the reader is familiar with npm, it should be clear what this means.

**Comment by ofrobots** [comment]
Ready for another look.

**Comment by matthewloring** [comment]
LGTM.

**Pull request merged**